construction/__manifest__.py

```diff
@@ -3,18 +3,17 @@
     'version': '1.0',
     'category': 'Construction',
     'depends': [
+        'base_automation',
```

Comment:
This will trigger the custo pricing (and I'm pretty sure we don't want that :-/ )